plugins/source/crowdstrike/client/client.go

```diff
@@ -0,0 +1,72 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+type Client struct {
+	logger   zerolog.Logger
+	spec     specs.Source
+	Services Services
+}
+
+type Services struct {
+	Incidents Incidents
+	Alerts    Alerts
+}
+
+func (*Client) Logger() *zerolog.Logger {
+	return &log.Logger
+}
+
+func (*Client) ID() string {
+	return "CrowdStrikeClient"
+}
+
+func New(ctx context.Context, logger zerolog.Logger, s specs.Source) (schema.ClientMeta, error) {
+	crowdStrikeSpec := &Spec{}
+	if err := s.UnmarshalSpec(&crowdStrikeSpec); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal CrowdStrike spec: %w", err)
+	}
+	clientId, ok := os.LookupEnv("FALCON_CLIENT_ID")
```

Comment:
Are these env vars already used in existing CS tools? If not, remove them or make them clearer in naming (`CROWDSTRIKE` instead of `FALCON` maybe)